QtCollider/widgets/QcWebView.h

```diff
@@ -58,13 +58,12 @@ class WebView : public QWebEngineView {
     Q_INVOKABLE void navigate(const QString& url);
 
 public Q_SLOTS:
-    void findText(const QString& searchText, bool reversed, QtCollider::QcCallback* cb);
+    void findText(const QString& searchText, bool reversed, QtCollider::QcCallback* cb = nullptr);
```

Comment:
I had left a reply under your other comment, but it was "pending" until short ago, sorry. I'll copy-paste:

I think the default there is nice, so that findText can also be called without a callback (see for instance HelpBrowser::findText in help_browser.cpp:255). What do you think?

Reply:
yeah this makes total sense!